libs/components/src/dialog/simple-configurable-dialog/simple-configurable-dialog.component.html

```diff
@@ -0,0 +1,24 @@
+<bit-simple-dialog>
+  <i bit-dialog-icon class="bwi tw-text-3xl" [class]="iconClasses" aria-hidden="true"></i>
+
+  <span bitDialogTitle>
+    {{ title }}
+  </span>
```

Comment:
We can keep this on a single line similar to `bitDialogContent`.

```suggestion
  <span bitDialogTitle>{{ title }}</span>
```